src/redux/localStorageUtils.ts

```diff
@@ -0,0 +1,88 @@
+export function createLocalStorageObject(name: string){
+    try {
+        let existing = localStorage.getItem(name);
+        if (!existing) {
+            setLocalStorageObject('notification', 'inProgressList', []);
+            setLocalStorageObject('notification', 'loadingList', []);
+            setLocalStorageObject('notification', 'erroredList', []);
+            setLocalStorageObject('notification', 'successList', []);
+            setLocalStorageObject('notification', 'isUpdate', "false");
+        }
+    }catch (error) {
+        console.log(error);
+    }
+};
+
+function setLocalStorageObject(name: string, key: string, value: any) {
+    let existing = localStorage.getItem(name);
+    let existingOne = existing ? JSON.parse(existing) : {};
+    existingOne[key] = value;
```

Comment:
@binduak it's kind of confusing with the variable name existing and existingOne can we rename?